src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -2,7 +2,6 @@
 
 import javafx.fxml.FXML;
 import javafx.scene.control.Label;
-import javafx.scene.layout.FlowPane;
```

Comment:
Sorry, just to confirm what was the FlowPane being used for, like how did it help in displaying tags? Is there any other part of our UI that we could use it for? But if not then yup can just delete.

Reply:
I think it was used for Tags, but we have removed it, so not removing this will cause checkstyle issues. 

Perhaps we can keep this in mind when we want to make the ClassGroup the blue button style UI instead? Id have to remove it for now to pass checkstyle unused imports